### ISSUE TITLE
Disable the scheduler for users when they upload a video

### ISSUE THREAD

**Issue by ruzakuku** [WWBN/AVideo#8232]
When a user wants to upload a video (through the encoder) he is offered to set a date and time for the video to be live. Is it possible to remove this feature, so that video will immediately be live?

![2023-07-24_195527](https://github.com/WWBN/AVideo/assets/38063103/db897801-fbee-4975-ac9f-c426e9b4da66)

**Comment by DanielnetoDotCom** [comment]
what about if you select "now"?

Even if you disable it here the Scheduler plugin still required in the streamer

**Comment by KrzysioMisio** [comment]
This plugin itself changes the time of video publication and you have to manually click in the admin panel to make the video visible.
This problem also returned at my place

**Comment by ruzakuku** [comment]
> what about if you select "now"?

Yes, but we don't want users to schedule their videos. "now" only :)

> Even if you disable it here the Scheduler plugin still required in the streamer

Good to know, so we will activate. But so, is it possible to disable the schedule?

**Comment by ruzakuku** [comment]
> This plugin itself changes the time of video publication and you have to manually click in the admin panel to make the video visible. This problem also returned at my place

We noticed that just uploaded video doesn't show up as the first one in the "most recent" ones. Maybe some bug here?

**Comment by DanielnetoDotCom** [comment]
https://github.com/WWBN/AVideo/issues/8205#issuecomment-1644390551
